src/Nethermind/Nethermind.Consensus/Processing/ParallelProcessing/Version.cs

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+namespace Nethermind.Consensus.Processing.ParallelProcessing;
+
+public readonly record struct Version(ushort TxIndex, ushort Incarnation)
+{
+    public bool IsEmpty => TxIndex == ushort.MaxValue && Incarnation == ushort.MaxValue;
```

Comment:
Go with default meaning 0? It'd be more natural.

Reply:
Transactions are indexed from 0...N, so special value is easier as MaxValue.
I might change it to int to support more than 65k transaction blocks, then special value can be -1.